script.js

```diff
@@ -0,0 +1,37 @@
+const header = document.querySelector('header');
+const section = document.createElement('section');
+header.appendChild(section);
+// section.innerHTML = `<p>Hello eorld</p>`;
+
+const open = document.querySelector('#menue-button');
+
+open.addEventListener('click', () => {
+  const window = document.createElement('div');
+  window.className = 'window';
+  const popup = document.createElement('div');
+  popup.className = 'popup';
+  popup.innerHTML = `<div class="all">
+  <i class="fa-solid fa-xmark close"></i>
+<ul id="popup">
+<li href="#portifolio">portifolio</li>
+<li>about</li>
+<li>contact</li>
+</ul>
+  </div>`;
+  window.appendChild(popup);
+  header.appendChild(window);
```

Comment:
- [ ] Please add the correct style for your mobile menu by positioning every element in the right place like the given design: 
- The close button suppose to be higher at the top and kindly make it bigger and add the correct color to it.
- and the nav links suppose to be lower than that.
So kindly fix that and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/168822799-89c70cee-30e6-4746-ae73-a2cd80861bfa.png)  | ![image](https://user-images.githubusercontent.com/68796232/168822529-a0d73d07-05aa-4696-a5e3-3ea05a043a51.png)  |